:sloth: Additional names for "vmware-harbor-registry"

adriens opened the pull request:
# :grey_question: About

Harbor VMWare Registry (aka. `Harbor`) has been added to eol:

- https://github.com/endoflife-date/endoflife.date/issues/4230

But when I tried to get its eols from its most commonly used name `harbor`, I got this:

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/05e706d8-dbc6-478e-8010-9fab3a3fc613)

Also, in littertaure it is better known as `Harbor`:

![image](https://github.com/endoflife-date/endoflife.date/assets/5235127/6cb26652-67cf-402c-b779-7c6fe13c0d8b)


# :dart: Goal of the PR

:point_right: By merging this PR, it will be **easier to get harbor eols wile using its most commonly used name.**

adriens committed:
Update vmware-harbor-registry.md

Add alternate names for Harbor

marcwrobel commented on products/vmware-harbor-registry.md:
```diff
@@ -4,6 +4,10 @@ category: server-app
 permalink: /vmware-harbor-registry
+alternate_urls:
+-   /harbor-registry
+-   /harbor
```
`/harbor` looks good to me, but I don't see the point in adding the other `alternate_urls` : they don't seem commonly used.

adriens committed:
Update vmware-harbor-registry.md

As per
https://github.com/endoflife-date/endoflife.date/pull/4339#discussion_r1433263646
